src/lib.rs

```diff
@@ -98,6 +88,19 @@
 //! At this time, we do not recommend use for security-critical applications.
 //! Use at your own risk.
 //!
+//! # Useful features
+//!
+//! The main interfaces allow a [`Participant`] to process messages from
+//! other participants, producing a set of outgoing messages and, if the
+//! protocol completed, an output.
```

Comment:
"The main interfaces" is doesn't mean much to me. Instead, I think we should just say something like "A `Participant`  `Participant`s. If ."

```suggestion
//! A [`Participant`] processes messages received from other [`Participant`]s and generates [`Message`] for other  [`Participant`]s to process. When the current sub-protocol finishes, output values are produced.
```